pinot-tools/pom.xml

```diff
@@ -268,6 +270,24 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!--
+      This dependency is needed for LaunchSparkDataIngestionJobCommand.
+      The dependency only contains a few classes and scala library which has been excluded.
+      Hence, it will not interfere with spark-core classes present in runtime env
+      and will use the env spark version to actually execute the spark job
+    -->
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-launcher_${scala.version}</artifactId>
+      <version>${spark.version}</version>
+      <exclusions>
```

Comment:
I do have some minor concerns about adding dependencies on Spark here. But since it's only pinot-tools, it's not so important.

Reply:
This dep is pretty basic. Only contains couple of classes and one more dependency. So including it isn't a main concern. Earlier I had included `spark-core` which was a major one and could have caused a lot of issues.